### DISCUSSION TITLE
An issue about pose calibration

### DISCUSSION THREAD

**Discussion by Django-Z** [jyjblrd/Mocap-Drones#35] [General]
Hi, when we clicked on the Collect points for camera pose calibration button, we used one infrared LED to move through the experimental space to collect points. However, the effect of camera pose calibration was poor. Can you please provide some specific operational details for camera pose calibration?
![2024-04-22 20-08-09 的屏幕截图](https://github.com/jyjblrd/Low-Cost-Mocap/assets/85657632/89bb227f-d9cc-4b52-9122-5477887daa4e)
![2024-04-22 20-08-39 的屏幕截图](https://github.com/jyjblrd/Low-Cost-Mocap/assets/85657632/a5c60ee2-dd84-486e-8fcd-ac54f8738f75)

**Comment by jyjblrd** [comment]
Hi, this discussion gives some tips on camera pose calibration and should hopefully help you: [https://github.com/jyjblrd/Low-Cost-Mocap/discussions/23#discussioncomment-8118097](https://github.com/jyjblrd/Low-Cost-Mocap/discussions/23#discussioncomment-8118097)

**Reply by Django-Z** [reply]
Thank you for your reply. I will carefully review that discussion!
